webapp/channels/src/components/suggestion/command_provider/command_provider.tsx

```diff
@@ -163,7 +163,7 @@ export default class CommandProvider extends Provider {
     handleMobile(pretext: string, resultCallback: ResultsCallback<AutocompleteSuggestion>) {
         const {teamId} = this.props;
 
-        const command = pretext.toLowerCase();
+        const command = pretext;
```

Comment:
This change makes it so commands are case sensitive (`/Away` won't trigger the `/away` command).

Is that intended?